speech-api/SpeechSynthesis-speak-without-activation-fails.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<p>Some body text.
+<script>
+var t = async_test('speechSynthesis.speak requires user activation');
+const utter = new SpeechSynthesisUtterance('1');
```

Comment:
Can you use the `async_test(f(n), name)` structure here as well? The reason is that if `new SpeechSynthesisUtterance('1')` throws an exception in the current structure, it would be a harness error plus a test that times out, instead of a failing test. Putting all the interesting code inside the first callback or in `t.step_func` for callbacks avoids this kind of problem.